downstream/titles/hub/managing-user-access/master.adoc

```diff
@@ -1,15 +1,19 @@
 = Managing user access in Private Automation Hub
 :imagesdir: images
 :numbered:
+include::attributes/attributes.adoc[]
 
-Configure user access in Automation Hub to provide the appropriate level of system permissions to groups in your organization.
+Configure user access in Automation Hub to provide the appropriate level of system permissions to groups in your organization, or provide view-only access to unauthorized users.
 
 include::hub/assembly-user-access.adoc[leveloffset=+1]
 
+include::hub/assembly-view-only-access.adoc[leveloffset=+1]
+
 == Conclusion
```

Comment:
Do you think this section is necessary ? (I know it's not strictly part of this PR)

Reply:
Do you mean the conclusion? It was there when I edited the document so I didn't really think to remove it

Reply:
Yes. I don't think the Conclusion section is needed. It's not in this PR's scope, though.

Reply:
I would agree with you. Let me take it out, thanks